tex/homology/singular.tex

```diff
@@ -747,7 +747,7 @@ \section{More examples of chain complexes}
 \begin{problem}
 	For $n \ge 1$ show that the composition
 	\[ S^{n-1} \injto D^{n} \taking{F} S^{n-1} \]
-	cannot be the identity map on $S^{n-1}$ for any $F$.
+	cannot be the identity map on $S^{n-1}$ for any continuous $F$.
```

Comment:
This one may be redundant (maps are implicitly assumed to be "topological space homomorphism" i.e. continuous throughout the book), but it confused me for a bit regardless. Feel free to rollback.

Reply:
i agree it's redundant but i think there's no reason not to add it (i often forget now because it's so obvious to me, haha).